### PR TITLE
DRILL-6198: OpenTSDB unit tests fail when Lilith client is run

### PR DESCRIPTION
**Pull request by vvysotskyi**
Added method which checks that the default port 10_000 is free, otherwise this method increases port number and checks until free port is found.

### CODE REVIEW

File: contrib/storage-opentsdb/src/test/java/org/apache/drill/store/openTSDB/TestOpenTSDBPlugin.java [hunk 1]
```diff
@@ -51,17 +54,17 @@
 
 public class TestOpenTSDBPlugin extends PlanTestBase {
 
-  protected static OpenTSDBStoragePlugin storagePlugin;
-  protected static OpenTSDBStoragePluginConfig storagePluginConfig;
+  private static int portNum = 10_000;
```
Comment (arina-ielchiieva) [inline contrib/storage-opentsdb/src/test/java/org/apache/drill/store/openTSDB/TestOpenTSDBPlugin.java]: Why do you set value right away? It looks you will always re-write in `setup`.

File: contrib/storage-opentsdb/src/test/java/org/apache/drill/store/openTSDB/TestOpenTSDBPlugin.java [hunk 2]
```diff
@@ -51,17 +54,17 @@
...
 
   @Rule
-  public WireMockRule wireMockRule = new WireMockRule(10000);
+  public WireMockRule wireMockRule = new WireMockRule(portNum);
 
   @BeforeClass
   public static void setup() throws Exception {
+    portNum = getFreePortNum(portNum, 1000);
```
Comment (arina-ielchiieva) [inline contrib/storage-opentsdb/src/test/java/org/apache/drill/store/openTSDB/TestOpenTSDBPlugin.java]: May be we can decrease number of attempt to 200?

File: contrib/storage-opentsdb/src/test/java/org/apache/drill/store/openTSDB/TestOpenTSDBPlugin.java [hunk 3]
```diff
@@ -185,4 +188,26 @@ public void testDescribe() throws Exception {
...
+   * Otherwise, increases port number and checks until free port is found
+   * or the number of attempts is reached specified numAttempts
+   *
+   * @param portNum     initial port number
+   * @param numAttempts max number of attempts to find port with greater number
+   * @return free port number
+   * @throws BindException if free port was not found and all attempts were used.
+   */
+  private static int getFreePortNum(int portNum, int numAttempts) throws IOException {
+    while (numAttempts > 0) {
```
Comment (arina-ielchiieva) [inline contrib/storage-opentsdb/src/test/java/org/apache/drill/store/openTSDB/TestOpenTSDBPlugin.java]: 1. Please re-write using for loop.
2. Please add more details to the exception, include initial port number, which ports were occupied. Suggest to check which ports are free etc.

File: contrib/storage-opentsdb/src/test/java/org/apache/drill/store/openTSDB/TestOpenTSDBPlugin.java (hunk 1 above)
Comment (vvysotskyi) [reply contrib/storage-opentsdb/src/test/java/org/apache/drill/store/openTSDB/TestOpenTSDBPlugin.java]: Thanks, removed.

File: contrib/storage-opentsdb/src/test/java/org/apache/drill/store/openTSDB/TestOpenTSDBPlugin.java (hunk 2 above)
Comment (vvysotskyi) [reply contrib/storage-opentsdb/src/test/java/org/apache/drill/store/openTSDB/TestOpenTSDBPlugin.java]: Done.

File: contrib/storage-opentsdb/src/test/java/org/apache/drill/store/openTSDB/TestOpenTSDBPlugin.java (hunk 3 above)
Comment (vvysotskyi) [reply contrib/storage-opentsdb/src/test/java/org/apache/drill/store/openTSDB/TestOpenTSDBPlugin.java]: 1. Thanks, it looks better with for loop.
2. Added more details to the error message.

**Comment by arina-ielchiieva** [comment]
+1, thanks for making the changes.

**Pull request closed without merging**